sample.txt

```diff
@@ -0,0 +1 @@
+This is my first commit 
```

Comment:
This is unnecessary remove this (let it be home house is not the right code changes)